### ISSUE TITLE
Amplify Broken

### ISSUE THREAD

**Issue by pbishnupuri** [aws-amplify/amplify-console#553]
** Please describe which feature you have a question about? **
Okay so I created everything using amplify init: this added auth, api, s3. However then I deleted the Cognito Userpool that amplify init created. This broke my amplify. Now when I go to my amplify console, click on my app, then click backend environments. I can click on everything (api, s3, functions, they all load) except for auth.

![image](https://user-images.githubusercontent.com/31415137/78097947-4dc76c80-73a3-11ea-88a1-aa6ccf3ae971.png)

It gets stuck like this.

** Provide additional details**

Currently my AWSAppsync API is connected to a userpool I created by hand (without the help of amplify). So can I just connect my Amplify Auth to that userpool? That's what I've been trying to do.

**Comment by swaminator** [comment]
@pbishnupuri can you please share your browser's console logs (right click > inspect element > Console)

**Comment by pbishnupuri** [comment]
Sorry about that @swaminator . What should I change the title to be? And is this what you were looking for for console?

![image](https://user-images.githubusercontent.com/31415137/78171685-75a5e700-741a-11ea-969e-1d5018999c3f.png)

**Comment by ganipcanot** [comment]
@pbishnupuri Can you please provide your appId and region so we can investigate further?

Under appSettings --> General --> AppArn it will look something like this:

arn:aws:amplify:region:accountId:apps/appId

Also if you can provide the UserPool Id that you manually created in your account.

**Comment by pbishnupuri** [comment]
For my cogito pool?
us-east-2
arn:aws:cognito-idp:us-east-2:199854827602:userpool/us-east-2_Psw3fjvcb
Pool Id: us-east-2_Psw3fjvcb

Will you be needing my amplify app id?

**Comment by ganipcanot** [comment]
@pbishnupuri Yes, but I think it is "d2hradfi81bvpl" from the screenshot you attached.  You can clarify if this is incorrect.

**Comment by pbishnupuri** [comment]
My AWS Amplify says this: 
amplify pull --appId d1bmr9l6bp7f36 --envName dev

So i'm not sure where your number is coming from.

**Comment by nimacks** [comment]
@pbishnupuri, is this still an issue for you?
